services/continuous_querier/service.go

```diff
@@ -353,6 +355,7 @@ func (s *Service) runContinuousQueryAndWriteResult(cq *ContinuousQuery) error {
 	// Execute the SELECT.
 	ch := s.QueryExecutor.ExecuteQuery(q, influxql.ExecutionOptions{
 		Database: cq.Database,
+		Quiet:    !s.logQueries,
```

Comment:
Hmmm... trying to think of a better name then for `logQueries` so that can be the default of false.  I can live with it as is though.